File: Java/Player.java
public class Player {
    
    static int id = 0;

    String username;
    int health;

    public Player(String username, int health) {
        this.username = username;
        this.health = health;
    }

    public void takeDamage(int damage) {
        this.health -= damage;
    }

    public String description() {
        return this.username + ":\nHealth: " + this.health;
    }
}

File: Python/player.py
class Player:
    def __init__(self, username):
        self.username = username
        self.health = 20

    def take_damage(self, damage):
        self.health -= damage


class Warrior(Player):
    def __init__(self, username, level):
        super().__init__(username)
        self.level = level
